index.html

```diff
@@ -1475,7 +1475,8 @@ <h2><dfn>SystemAudioPreferenceEnum</dfn>
 
         <p>Describes whether an application invoking
         {{MediaDevices/getDisplayMedia()}} would like the user agent to include
-        system audio among the audio sources offered to the user.</p>
+        system audio among the audio sources offered to the user for
+        [=display surface/monitor=] [=display surfaces=].
```

Comment:
Addressing @martinthomson's comment.
```suggestion
        [=display surface/monitor=] [=display surfaces=]. Does not apply to any
        other type of [=display surface=].
```